.assets/readme_chapters/FAQs.md

```diff
@@ -0,0 +1,35 @@
+# FAQs
+
+## General
+* You will not see the non-Java frames _preceding_ the Java frames on the
+  stack, unless `--cstack vm` is specified.
```

Comment:
There'll be a separate chapter about stack walking. It will describe all `cstack` options, a bit of implementation detail, and related limitations (like MaxJavaStackTraceDepth mentioned below).

Reply:
FAQ removed